### PR TITLE
CNV_3044_3 Add Disk to VM files specific to 4.2 branch

### PR DESCRIPTION
**Pull request by bgaydosrh**
Checking build will comment later

### CODE REVIEW

**Comment by bgaydosrh** [comment]
The content in this PR has been code reviewed and QE'd.

It does need a peer review then merge and pick to enterprise-4.2 ONLY.

Please label **Peer Review Needed** and **Enterprise-4.2**.

Thanks,
Bob

**Comment by bgaydosrh** [comment]
Test Build link: http://file.bos.redhat.com/bgaydos/121719_2/cnv/cnv_users_guide/

File: cnv/cnv_users_guide/cnv-editing-vm-template.adoc [hunk 1]
```diff
@@ -6,6 +6,5 @@ toc::[]
 
 You can edit a virtual machine template in the web console.
 
-include::modules/cnv-editing-template-web.adoc[leveloffset=+1]
-
-
+include::modules/cnv-editing-vm-yaml-web.adoc[leveloffset=+1]
```
Comment (aburdenthehand) [inline cnv/cnv_users_guide/cnv-editing-vm-template.adoc]: This should be `cnv-editing-template-web.adoc. The functionality of `cnv-editiing-vm-yaml-web` is not available in 2.1

Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-editing-vm-template.adoc]: Done.

**Comment by bgaydosrh** [comment]
This PR is ready for final Peer Review and merge to enterprise-4.2 ONLY.

**Comment by bgaydosrh** [comment]
Test Build: http://file.bos.redhat.com/bgaydos/122119_5/cnv/cnv_users_guide/

File: modules/cnv-add-disk-to-vm.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+= Adding a virtual disk to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Disks* tab.
+. Click *Add Disks* to open the *Add Disk* window.
+. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+. Use the drop-down lists and check boxes to edit the disk configuration.
```
Comment (ousleyp) [inline modules/cnv-add-disk-to-vm.adoc]: I think this step could be rewritten to be clearer.

Nit: I think this should start with what you're actually doing and end with how you're doing it ("Edit the disk configuration by _____"). 

Nit 2: "Use the drop-down lists and check boxes" is vague. I think "edit the disk configuration" *might* also be too vague.

**Review by ousleyp — COMMENTED** [review]
LGTM other than one comment.

File: modules/cnv-add-disk-to-vm.adoc (hunk 2 above)
Comment (bgaydosrh) [reply modules/cnv-add-disk-to-vm.adoc]: Sent @ousleyp an email to clarify. Will follow-up on 1/7/20.

Comment (bgaydosrh) [reply modules/cnv-add-disk-to-vm.adoc]: Pan said "The first comment was just about reordering the phrase, so you could even say "Edit the disk configuration by using the drop-down lists and check boxes."

I will do this.

Comment (bgaydosrh) [reply modules/cnv-add-disk-to-vm.adoc]: Done. Also added intro sentence to procedure.

**Pull request merged**